src/functionalTest/resources/features/F-003 - Get Translations/S-003.1.td.json

```diff
@@ -0,0 +1,34 @@
+{
+  "_guid_": "S-003.1",
+  "_extends_": "Translation_Cy_Base",
+  "title": "must return a successful response when translate operation is requested",
+  "specs": [
+    "an active solicitor profile"
+  ],
+  "users": {
+    "invokingUser": {
+      "_extends_": "BeftaCaseworker2Solicitor2"
+    }
+  },
+  "request": {
+    "_extends_": "Common_Request",
+    "headers": {
+      "Accept": "application/json"
+    },
```

Comment:
(for action under WLTS-16)
This part of the request can proabbly be moved to `Translation_Cy_Base` file.  (same for other scenarios)

Reply:
Withdrawn as none of the other base endpoint TD files follow this formatting either.